test/test-watch-cpu-profile.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+var agent = require('../lib/agent')();
+var tap = require('tap');
+var w = require('./watcher');
+var cpu = require('../lib/watcher/cpu-profile');
+
+console.assert(agent);
```

Comment:
It is, but leaving the code wasn't intentional, though not harmful, now removed.